### PR TITLE
[core] chomp, not strip output in CommandExecutor

### PR DESCRIPTION
**Pull request by dotdoom**
### Checklist
- [x] #15131 is merged
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

Current output from CommandExecutor is hard to parse for a human eye when indentation is important.

### Description

[`String#strip`](https://ruby-doc.org/core-2.6/String.html#method-i-strip) removes whitespaces (and CR/LF) *around* the line, while [`String#chomp`](https://ruby-doc.org/core-2.6/String.html#method-i-chomp) removes only record separator (CR/LF) at the end of the line.

### CODE REVIEW

**Comment by joshdholtz** [comment]
@dotdoom #15131 is now merged if you would like to rebase this 😊

**Comment by dotdoom** [comment]
@joshdholtz done, thank you!

**Review by joshdholtz — APPROVED** [review]
This is so good 😊 Thanks for catching this ❤️
